src/main/deploy/pathplanner/autos/New Auto.auto

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Maybe give this file a more meaningful name?

Reply:
I removed this one. Done